apps/frontend/src/pages/meeting-details/meeting-details.tsx

```diff
@@ -97,7 +97,16 @@ const MeetingDetails: React.FC = () => {
 					meeting.id,
 				);
 				const host = config.ENV.APP.HOST;
-				void navigator.clipboard.writeText(`${host}${publicUrl}`);
+				const fullUrl = `${host}${publicUrl}`;
+
+				const textarea = document.createElement("textarea");
+				textarea.value = fullUrl;
+				document.body.append(textarea);
+				textarea.select();
```

Comment:
Just in case, move textarea outside of visible view and make its opacity 0. I know we remove it, but just in case

Reply:
Yes, sure. Smth like `textarea.style.opacity = 0`. Same for moving outside of view